test/src/test/java/org/corfudb/integration/CorfuReplicationMultiSourceSinkIT.java

```diff
@@ -90,10 +90,26 @@ public class CorfuReplicationMultiSourceSinkIT extends AbstractIT {
     // Listens to replication status updates on a Sink cluster
     private List<ReplicationStatusListener> replicationStatusListeners = new ArrayList<>();
 
-    protected void setUp(int numSourceClusters, int numSinkClusters) throws Exception {
+    protected void setUp(int numSourceClusters, int numSinkClusters, ExampleSchemas.ClusterUuidMsg topologyType) throws Exception {
```

Comment:
minor: since we pass the topology type, numSourceClusters and numSinkClusters can be inferred from it?

Reply:
No, we cannot infer it as we don't have access to the topology itself.
The only way we can infer is to have if-else statements to check the topology type , but passing it explicitly seems cleaner and efficient. 

Please let me know if you think otherwise. Open to discuss it.